jb/fftw/tde_result.hpp

```diff
@@ -0,0 +1,122 @@
+#ifndef jb_fftw_tde_result_hpp
+#define jb_fftw_tde_result_hpp
+
+#include <jb/detail/array_traits.hpp>
+#include <jb/fftw/aligned_multi_array.hpp>
+
+namespace jb {
+namespace fftw {
+
+/**
+ * tde_result is a type to handle the result of operating over the
```

Comment:
ACK. working on it. I will add the documentation to other operations like sum2 (or the average, in the future when we implement Normalization)